src/scss/packages/_uswds.scss

```
@@ -4,3 +4,4 @@
 // Custom Login.gov Design System components
 @forward 'usa-code/src/styles';
 @forward 'usa-verification-badge/src/styles';
+@forward 'usa-image-example';
```

Nit: Even just for consistency, I'd reference `/src/styles` directly like the others.

```suggestion
@forward 'usa-image-example/src/styles';
```

The difference being that the main package entrypoint would potentially have dependencies (it doesn't in our case), and loading those dependencies would be redundant from the main `uswds` package, since they're already expected to be included. The redundant loading can have a performance impact (see also https://github.com/uswds/uswds/pull/5241#issuecomment-1540686386).

---

Got it. Making this change is basically telling sass to switch from:
- `@forward 'usa-image-example';` - Include the component styles and all dependencies.
- `@forward 'usa-image-example/src/styles';` - Include the component styles, but not dependencies.

---

Yep, exactly. If we needed dependencies, we could add them to `src/scss/packages/_usa-image-example.scss`†, and because this file `_uswds.scss` includes _everything_, we should be able to confidently assume we'd not need those dependencies to be reloaded, and therefore we can improve performance by referencing the styles directly.

----

† This is what dependencies look like in a USWDS component, for example:

https://github.com/uswds/uswds/blob/develop/packages/usa-accordion/_index.scss